packages/datadog-instrumentations/src/undici.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const {
+  addHook
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const tracingChannel = require('dc-polyfill').tracingChannel
+const ch = tracingChannel('apm:undici:fetch')
+
+const { createWrapFetch } = require('./helpers/fetch')
+
+addHook({
+  name: 'undici',
+  versions: ['^4.4.1', '5', '^6.0.0']
```

Comment:
The last range should be `>=6` so that tracing will continue to work if a user upgrades to 7.0 when it's released.